### PR TITLE
feat(builtins): json.obj_pairs, json.puts_pairs to build JSON objects from array of pairs

### PR DESCRIPTION
**Pull request by folex**
`json.obj` and `json.puts` are basically varargs, which gives flexibility, but is impossible to express in well-typed Aqua.

`json.obj` allows to build JSON object from an array of `(string, json)` pairs.

### CODE REVIEW

File: particle-builtins/src/builtins.rs [hunk 1]
```diff
@@ -255,6 +255,10 @@ where
             ("json", "puts")       => wrap(json::puts(args)),
             ("json", "parse")      => unary(args, |s: String| -> R<JValue, _> { json::parse(&s) }),
             ("json", "stringify")  => unary(args, |v: JValue| -> R<String, _> { Ok(json::stringify(v)) }),
+            ("json", "obj_pairs")  => unary(args, |vs: Vec<(String, JValue)>| -> R<JValue, _> { json::obj_from_pairs(vs) }),
+            ("json", "puts_pairs") => binary(args, |obj: JValue, vs: Vec<(String, JValue)>| -> R<JValue, _> { json::puts_from_pairs(obj, vs) }),
+
+            ("run-console", "print") => wrap_unit(Ok(log::debug!(target: "run-console", "{}", json!(args.function_args)))),
```
Comment (kmd-fl) [inline particle-builtins/src/builtins.rs:261]: Do you forget to remove it? Afaik we always remove such builtins after debugging.

Comment (folex) [reply particle-builtins/src/builtins.rs:261]: nope, it's for support of `run-console` in Aqua. `fluence-js` supports it, don't see a reason why `rust-peer` shouldn't

Comment (kmd-fl) [reply particle-builtins/src/builtins.rs:261]: Maybe some wrongdoer could try to pollute our logs even more? Could it make more sense to allow `run-console.print` only for management_id?

Comment (folex) [reply particle-builtins/src/builtins.rs:261]: That wrongdoer would have to enable `debug` for `run-console`, eg pass variable `RUST_LOG=run-console=debug`. 

I think that's a good-enough protection. WDYT?

Comment (kmd-fl) [reply particle-builtins/src/builtins.rs:261]: Yeah, I didn't notice, sry

Comment (folex) [reply particle-builtins/src/builtins.rs:261]: no worries! it's good to raise concerns ^_^

**Review by justprosh — APPROVED** [review]
lgtm, but it will be nice to have these new methods called at least once in the tests

**Pull request merged**